packages/edit-site/src/components/navigation-sidebar/navigation-toggle/index.js

```diff
@@ -1,12 +1,12 @@
 /**
  * WordPress dependencies
  */
-import { useSelect } from '@wordpress/data';
+import { useDispatch, useSelect } from '@wordpress/data';
 import { Button, Icon } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { wordpress } from '@wordpress/icons';
 
-function NavigationToggle( { icon, isOpen, onClick } ) {
```

Comment:
since we no longer pass the `onClick` handler through, should we also get `isOpen` from the state directly here instead of via props? (not sure myself, up for discussion)

Reply:
`NavigationToggle`'s direct parent, `NavigationSidebar` also uses the isOpen prop (which resolves to isNavigationOpened). It felt odd having identical selectors in both the parent and child, but I don't have a strong opinion here.